server/auth/sso/jwt_test.go

```diff
@@ -14,27 +14,27 @@ const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwi
 
 func TestClaimSetFor(t *testing.T) {
 	t.Run("Empty", func(t *testing.T) {
-		claimSet, err := ClaimSetFor(&rest.Config{})
+		claims, err := ClaimSetFor(&rest.Config{})
 		if assert.NoError(t, err) {
-			assert.Nil(t, claimSet)
+			assert.Nil(t, claims)
 		}
 	})
 	t.Run("Basic", func(t *testing.T) {
-		claimSet, err := ClaimSetFor(&rest.Config{Username: "my-username"})
+		claims, err := ClaimSetFor(&rest.Config{Username: "my-username"})
 		if assert.NoError(t, err) {
-			assert.Empty(t, claimSet.Iss)
-			assert.Equal(t, "my-username", claimSet.Sub)
+			assert.Empty(t, claims.Issuer)
+			assert.Equal(t, "my-username", claims.Subject)
```

Comment:
It likely won't matter, but `iss` and `sub` are the standard names for these fields so future tooling might will have opinions about the naming of these claims.